src/main/java/org/kohsuke/github/GHGistFile.java

```diff
@@ -60,11 +64,16 @@ public String getLanguage() {
     }
 
     /**
-     * Content of this file.
+     * Content of this file. Lazy download the content in case the size is nonzero but the content is empty. Function
+     * modified for <a href="https://github.com/hub4j/github-api/issues/1325">issue #1325</a>.
      *
      * @return the content
      */
-    public String getContent() {
+    public String getContent() throws IOException {
+        if (this.content == null && this.size > 0) {
+            InputStream inputStream = new URL(this.raw_url).openStream();
+            this.content = new String(inputStream.readAllBytes());
```

Comment:
This works while testing, but this circumvents the `GitHubClient`.  If the user has configured proxying or expects to use a different http client library, this will not function the way they want.  Please use the the `Requester` class like all other parts of the library.

Reply:
https://github.com/hub4j/github-api/blob/06b031278894f3a1a5ea804c2720e0e7e0baaf63/src/main/java/org/kohsuke/github/Requester.java#L116

Reply:
Also, if the size is 0, we should return `""` right?